scripts/leapp_preupgrade.py

```diff
@@ -269,93 +350,27 @@ def parse_results(output):
 
 def call_insights_client():
     print("Calling insight-client in background for immediate data collection.")
-    run_subprocess(["insights-client"], wait=False)
+    run_subprocess(["/usr/bin/insights-client"], wait=False)
```

Comment:
We can reuse it from [https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/conversion_script.py#L414](https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/conversion_script.py#L414 ) as you suggested.